app/models/notification.rb

```diff
@@ -10,19 +10,23 @@
 #  updated_at      :datetime         not null
 #  account_id      :bigint(8)        not null
 #  from_account_id :bigint(8)        not null
+#  type            :string
 #
 
 class Notification < ApplicationRecord
+  self.inheritance_column = nil
```

Comment:
What does this mean?

Reply:
ActiveRecord uses “type” column to typecast records into subclasses. This disables that. See also in media attachment class.